src/desktop/apps/fair_organizer/stylesheets/index.styl

```diff
@@ -184,10 +184,9 @@ browse-container-height = 442px
 .fair-organizer-top__countdown__clock .clock-header
   border 0
   padding 0
-  margin 20px 0
+  margin 20px 0 0 0
```

Comment:
nbd but could be `getSpace` as elsewhere?

Reply:
🥴 I didn't `require` palette here and now I'm afraid of angering the build again; we'll let it ride. 😉